aten/src/ATen/native/native_functions.yaml

```diff
@@ -176,6 +176,17 @@
   dispatch:
     CUDA: masked_scale_cuda
 
+- func: native_dropout(Tensor input, float p, bool? train) -> (Tensor, Tensor)
```

Comment:
why is `train` argument optional? It's explicitly specified at all callsites.

Reply:
This is trying to make it more like the old `_fused_dropout` API. I think the motivation is to make it an easy drop-replace thing. I could have misunderstood the comment here: https://github.com/pytorch/pytorch/pull/63937#discussion_r738270661

Reply:
Ah I see, but you are not replacing it. Ok, nevermind then.